user-manual.md

```diff
@@ -0,0 +1,260 @@
+
+restic is written in the Go programming language. At the moment, the only way to install restic on your system is to compile it from source. You need at least Go version 1.3.
+
+In order to build restic, execute the following steps:
+
+{% highlight console %}
+$ git clone https://github.com/restic/restic
+[...]
+
+$ cd restic
+
+$ go run build.go
+[...]
+
+$ ./restic --help
+Usage:
+  restic [OPTIONS] <command>
+
+Application Options:
+  -r, --repo= Repository directory to backup to/restore from
+
+Help Options:
+  -h, --help  Show this help message
+
+Available commands:
+  backup     save file/directory
+  cache      manage cache
+  cat        dump something
+  find       find a file/directory
+  fsck       check the repository
+  init       create repository
+  key        manage keys
+  list       lists data
+  ls         list files
+  restore    restore a snapshot
+  snapshots  show snapshots
+  version    display version
+{% endhighlight %}
+
+## <a name="initialize-repository"></a>Initialize a repository
+
+First, we need to create a "repository". This is the place where your backups will be saved at.
+
+In order to create a repository at `/tmp/backup`, run the following command and enter the same password twice:
+
+{% highlight console %}
+$ restic init --repo /tmp/backup
+enter password for new backend:
+enter password again:
+created restic backend 085b3c76b9 at /tmp/backup
+Please note that knowledge of your password is required to access the repository.
+Losing your password means that your data is irrecoverably lost.
+{% endhighlight %}
+
+Remembering your password is important! If you lose it, you won't be able to access data stored in the repository.
+
+## <a name="create-snapshot"></a>Create a snapshot
+
+Now we're ready to backup some data. Run the following command and enter the repository password you chose above again:
+
+{% highlight console %}
+$ restic -r /tmp/backup backup ~/work
+enter password for repository:
+scan [/home/user/work]
+scanned 764 directories, 1816 files in 0:00
+[0:29] 100.00%  54.732 MiB/s  1.582 GiB / 1.582 GiB  2580 / 2580 items  0 errors  ETA 0:00
+duration: 0:29, 54.47MiB/s
+snapshot 40dc1520 saved
+{% endhighlight %}
+
+As you can see, restic created a backup of the directory and was pretty fast! If you run the command again, restic will create another snapshot of your data, but this time it's even faster. This is deduplication at work! 
+
+{% highlight console %}
+$ restic -r /tmp/backup backup ~/shared/work/web
+enter password for repository:
+using parent snapshot 40dc1520aa6a07b7b3ae561786770a01951245d2367241e71e9485f18ae8228c
+scan [/home/user/work]
+scanned 764 directories, 1816 files in 0:00
+[0:00] 100.00%  0B/s  1.582 GiB / 1.582 GiB  2580 / 2580 items  0 errors  ETA 0:00
+duration: 0:00, 6572.38MiB/s
+snapshot 79766175 saved
+{% endhighlight %}
+
+You can even backup individual files in the same repository. 
+
+{% highlight console %}
+$restic -r /tmp/backup backup ~/work.txt
```

Comment:
Add a space between the `$` and `restic`, the same goes for all the other sample command lines.